js/intro.js

```diff
@@ -77,6 +77,29 @@ function generateSavedDetailsForm(users) {
   const container = createElement({ type: 'div', classList: ['container'] });
   renderIntroPage.appendChild(container);
 
+  const githubCreatedValue = createElement({
+    type: 'h4',
+    classList: [
+      `${dateDiff.years > 0 ? 'github-created-text' : 'github-created-alert'}`,
+    ],
+  });
+
+  if (dateDiff.years > 0) {
+    githubCreatedValue.innerText = `User GitHub account created ${dateDiff.years} years ago`;
+  } else if (dateDiff.years === 0 && dateDiff.months > 0) {
+    githubCreatedValue.innerText = `User GitHub account created ${dateDiff.months} months ago`;
+  } else if (
+    dateDiff.years === 0 &&
+    dateDiff.months === 0 &&
+    dateDiff.days >= 0
+  ) {
+    githubCreatedValue.innerText = `User GitHub account created ${
+      dateDiff.days > 0 ? dateDiff.days : '1'
+    } days ago`;
+  }
+
+  container.appendChild(githubCreatedValue);
+
```

Comment:
Can you test the classes and page UI as puppeteer can test this

Reply:
This is not a user interface, which means user is not able to click anywhere it's just a text

Reply:
puppeteer can test the text so test the text also we are already testing the test and classes in dashboard site

Reply:
I have created a new ticket for this

Reply:
it's already mentioned